pkg/registry/ggcr/client.go

```diff
@@ -51,7 +51,10 @@ type client struct {
 	writeRemoteIndex indexWriter
 }
 
-var _ RegistryClient = &client{}
+var (
+	_ RegistryClient = &client{}
+	_ registry.Client = &client{}
```

Comment:
maybe add a comment to explain why these are initialized here

Reply:
ok